src/Tests/Microsoft.NET.Build.Containers.IntegrationTests/TargetsTests.cs

```diff
@@ -131,19 +131,19 @@ public void ShouldNotIncludeSourceControlLabelsUnlessUserOptsIn(bool includeSour
     [InlineData("7.0.100-rc.1", "v7.0", "7.0")]
     [InlineData("8.0.100", "v8.0", "8.0")]
     [InlineData("8.0.100", "v7.0", "7.0")]
-    [InlineData("8.0.100-preview.7", "v8.0", "8.0-preview.7")]
-    [InlineData("8.0.100-rc.1", "v8.0", "8.0-rc.1")]
+    [InlineData("8.0.100-preview.7", "v8.0", "8.0.0-preview.7")]
+    [InlineData("8.0.100-rc.1", "v8.0", "8.0.0-rc.1")]
     [InlineData("8.0.100-rc.1", "v7.0", "7.0")]
     [InlineData("8.0.200", "v8.0", "8.0")]
     [InlineData("8.0.200", "v7.0", "7.0")]
     [InlineData("8.0.200-preview3", "v7.0", "7.0")]
     [InlineData("8.0.200-preview3", "v8.0", "8.0")]
     [InlineData("6.0.100", "v6.0", "6.0")]
     [InlineData("6.0.100-preview.1", "v6.0", "6.0")]
-    [InlineData("8.0.100-dev", "v8.0", "8.0-preview")]
-    [InlineData("8.0.100-ci", "v8.0", "8.0-preview")]
-    [InlineData("8.0.100-alpha.12345", "v8.0", "8.0-preview.1")]
-    [InlineData("9.0.100-alpha.12345", "v9.0", "9.0-preview.1")]
+    [InlineData("8.0.100-dev", "v8.0", "8.0.0-preview")]
+    [InlineData("8.0.100-ci", "v8.0", "8.0.0-preview")]
+    [InlineData("8.0.100-alpha.12345", "v8.0", "8.0-preview")]
+    [InlineData("9.0.100-alpha.12345", "v9.0", "9.0-preview")]
```

Comment:
Is it correct? Should it be 8.0.0-preview instead?

Reply:
oddly no - the 8.0-preview floating tag continues to exist and float across the different `8.0.0-preview.N` tags as new previews are released.